client/asset/interface.go

```diff
@@ -161,6 +161,8 @@ type Balance struct {
 	Immature uint64 `json:"immature"`
 	// Locked is the balance that that is currently locked for swap, but
 	// not actually swapped yet.
+	// TODO: this may actually be more than the balance locked for swap,
+	// if user manually locks some coins for example.
 	Locked uint64 `json:"locked"`
```

Comment:
Should we just update this comment to reflect reality - `total amount locked in the wallet which includes but is not limited to funds locked for swap, but not actually swapped yet`; and probably rename to `TotalLocked`.

My primary concern is that `xcWallet.balance` now reports the following locked balances, which can easily become confusing:
- ContractLocked (not _actually_ locked in the wallet)
- OrderLocked (locked in wallet but for orders only)
- Locked (total locked in wallet which may include funds not locked for orders)

OrderLocked is included in Locked, but Locked (which should be total locked amount) does not include ContractLocked.

I think it'll be less confusing to just split everything out into separate buckets:
- ContractLocked
- OrderLocked
- OtherLocked or UnknownLocked

Alternatively, since we now have ContractLocked and OrderLocked computed for each market, could we do away with the xcWallet.balance.OrderLocked and xcWallet.balance.ContractLocked balances and just keep xcWallet.balance.Locked, which is a reflection of the amount actually locked in the wallet.